apps-rendering/src/client/article.ts

```diff
@@ -80,26 +79,30 @@ function followToggle(
 ): void {
 	const followStatus = document.querySelector(querySelector);
 	if (!followStatus) return;
-	void bridgetClient.isFollowing(topic).then((following) => {
-		if (following) {
-			void bridgetClient.unfollow(topic).then((_) => {
-				ReactDOM.render(
-					h(followStatusComponent, {
-						isFollowing: false,
-						contributorName: topic.displayName,
-					}),
-					followStatus,
-				);
+	void bridgetClient.isFollowing(topic).then((isFollowing) => {
+		if (isFollowing) {
+			void bridgetClient.unfollow(topic).then((unfollow) => {
```

Comment:
I would perhaps call this `unfollowRes` to make it clear it is the response from the client rather than the state of `unfollow`